tests/robot-tests/tests/general_public/methodology_page_absence.robot

```diff
@@ -19,12 +19,12 @@ Navigate to Pupil absence in schools in England methodology page
 Should only be one link to absence methodology page
     [Documentation]  DFE-1359
     [Tags]  HappyPath
-    ${count}=  get element count  xpath://a[text()="Pupil absence statistics: methodology"]
+    ${count}=  get element count  xpath://ul/h3[text()="Pupil absence statistics: methodology"]
     should be true   ${count} == 1
 
 User navigates to absence methodology page
     [Tags]  HappyPath
-    user clicks link    Pupil absence statistics: methodology
+    user clicks element     xpath://ul/h3[text()="Pupil absence statistics: methodology"]/../div/a[text()="View methodology"]
```

Comment:
Can you make use of the Python changes you already made for "page contains methodology link" here too?

Reply:
I've created a new "user clicks methodology link" keyword to abstract this away, and added a "user checks methodology link" check before I call it